labs/a1.md

```diff
@@ -165,6 +202,47 @@ use [shell parameter expansion][1].
 
 [1]: https://stackoverflow.com/questions/965053/extract-filename-and-extension-in-bash
 
+# Question 3
+
+At some point, everyone has looked at a problem and thought to themselves:
+"Hey, I can do this in one line!"
+
+Lets find out if you can. I need to sort out some of my most listened to albums
+by making directories for each of them, specifically for my favorite artist Future.
+
+I have hosted a list of my favorite albums followed by their respective artist at 
+`https://raw.githubusercontent.com/0xcf/decal-labs/master/a1/albums.txt`,
+in a comma delimited format like `Die Lit, Playboi Carti`. For the GOAT artist Future,
+I want to create a folder for each of his albums. For example, for an entry like
+`SUPER SLIMEY, Future` I would expect a directory called `SUPER SLIMEY` to be created.
+
+**TLDR**: You need to fetch the list from the web, filter out the albums we want,
+trim out the album name, and then make a directory for each one, all in one line.
+
+```bash
+$ cat albums.txt
+...
+Drip or Drown 2, Gunna
+Playboi Carti, Playboi Carti
+DS2 (Deluxe), Future  <-- GOAT album detected!
+Drip Harder, Lil Baby
+The WIZRD, Future  <-- GOAT album detected!
+What a Time To Be Alive, Drake
+...
+
+# After our magic one liner...
+
+$ ls
+'DS2 (Deluxe)'   'The WIZRD'   ...
+# We got our new directories!
+```
+
+**Hints:** 
+- What common text manipulation commands can help you solve this?
+- As always, be aware that there isn't one unique solution to this problem!
+
+Submit your one line solution on Gradescope!
+
```

Comment:
maybe we can add a suggested skeleton for this one-liner question? something like `cat ______ | grep _______ | sed _______ | tr _____________ | xargs _____________`? (just make sure we're clear this is only a suggestion, and not required to follow)